tools/cli/commands/changelog.js

```diff
@@ -743,6 +743,17 @@ async function promptChangelog( argv, needChangelog, types ) {
 		},
 	} );
 
+	let userFacing = false;
+	// Ask if this is a user facing change if we detect Jetpack.
+	if ( argv.project === 'plugins/jetpack' ) {
+		userFacing = await prompt( {
+			type: 'confirm',
+			name: 'userFacing',
+			message: 'Is this a Jetpack change that site admins would like to know about? (Y/n)',
```

Comment:
What if we made this even more explicit about the end goal?

```suggestion
			message: 'Is this a change that should be mentioned in the release post? (Y/n)',
```
Or something of the sort?

Reply:
Personally I find the "site admins" question a little clearer than the other, but this is the kind of question where I don't necessarily trust my own opinion.